wit/runtime_config.wit

```diff
@@ -1,16 +1,21 @@
-package wasi:config@0.1.0;
+package wasi:runtime-config@0.1.0;
 
-interface runtime {
+interface config {
     /// Errors that can be returned from config sources
-    variant config-error {
+    variant error {
         /// An error occurred on the config source when fetching data
         upstream(string),
         /// I/O or connection failure when fetching data
         io(string),
-    };
+    }
 
     /// Gets a single opaque config value set at the given key if it exists
-    get: func(key: string) -> result<option<list<u8>>, config-error>;
+    get: func(key: string) -> result<option<list<u8>>, error>;
     /// Gets a list of all set config data
-    get-all: func() -> result<list<tuple<string, list<u8>>>, config-error>;
+    get-all: func() -> result<list<tuple<string, list<u8>>>, error>;
 }
+
+world imports {
```

Comment:
I also dropped having a `world.wit` because it wasn't needed. Is this what all the interfaces are doing now? Just want to make sure we don't add it unless we need to

Reply:
Yeah the primary reason that I added a `world` is because it follows the same patterns as other wasi repos. There is a `imports` world where you could just `include wasi:config/imports` to form bigger worlds. An use case is in `wasi:cloud-core`